examples/import_glance_image.py

```diff
@@ -56,7 +56,7 @@
 # image we need to retrieve all of them and do the filtering explicitly:
 images = images_service.list()
 image = next(
-    (i for i in images if i.name == 'CirrOS 0.3.4 for x86_64'),
+    (i for i in images if i.name == 'CirrOS 0.4.0 for x86_64'),
```

Comment:
I updated this because 0.3.4 isn't available in a default install any more, but 0.4.0 is.

Reply:
It would be better to extract the image name to a constant in the top of the example or make it an argument to the script.

Reply:
I agree, that's a good idea. I just didn't want to change it too much. But I think I can squeeze that in ;-)